sqs-purge.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+import axios from "axios";
+import fs from "fs";
+import { parse } from "yaml";
+
+
+const purgeQueues = async () => {
```

Comment:
Is `purgeQueues` only for local dev and testing? If so, shall we add the `isNodeEnvProd()` here for safety checking?

Reply:
It is reading data from docker-compose and explicitly reading URLs from it (which target to some magical hostname); I don't think it would do any harm in prod, even if someone manages to invoke it.

But shall we rename it maybe to be explicit a little bit? `sqs-purge-elasicmq-queues` or something like that?